### DISCUSSION TITLE
tags/

### DISCUSSION THREAD

**Discussion by giscus** [kuaimiaoda/Giscus_Comment#2] [Announcements]
# tags/

仓鼠在嘴里塞满了吃的去找龙猫医生。龙猫医生说:“你上火了呀，脸都肿了，我给你开点药吧。”仓鼠托着脸说:“我没有事的。我就是给你带点好吃的。” 大概就是这样，在不爱你的人眼里，你的喜欢是种病。

https://kuaimiaoda.github.io/tags/

**Comment by 20040122** [comment]
好
